practitioner-service/src/test/java/uk/ac/ox/ndph/mts/practitioner_service/service/PractitionerServiceTests.java

```diff
@@ -116,23 +119,46 @@ void TestSavePractitioner_WhenInvalidPractitioner_DoesntSavesToStore() {
     @Test
     void TestPractitionerService_WhenNullValues_ThrowsInitialisationError() {
```

Comment:
name is incorrect now.
Also, I think we should check with the others if we want to keep our own exception here, or a generic NPE. Perhaps ask on the channel?

Reply:
making sure with others in the channel that we go here for NPE as it is not in the controller level

Reply:
Looks like we are good here according to the discussion in the channel because this is a service class and not a controller class.